aten/src/ATen/core/TensorAccessor.h

```diff
@@ -156,7 +158,9 @@ class GenericPackedTensorAccessorBase {
   }
 protected:
   PtrType data_;
+  // NOLINTNEXTLINE(*c-arrays*)
```

Comment:
Actually why don't we use C++ std::array here? Do we need them in some constexpr context? Might be worth updating them.

Reply:
I tired, but the constructor depends on the ability to decay C-array into pointers so the related class hierarchy and other places need to be changed too. Too many specific changes mean they should be put in a later PR.